apps/web/src/stores/documentLogsWithEvaluationResults.ts

```diff
@@ -31,10 +33,16 @@ export function useDocumentLogsWithEvaluationResults(
       .evaluations.detail({ evaluationId }).logs.root,
     {
       serializer: (rows) => rows.map(documentLogPresenter),
+      searchParams: compactObject({
+        page: page ? String(page) : undefined,
+        pageSize: pageSize ? String(pageSize) : undefined,
+      }) as Record<string, string>,
```

Comment:
🐛 Fixed bug where document logs pagination where broken because page was not passed to the API